StudyBuddy/app/src/main/java/uw/studybuddy/LoginAndRegistration/LoginActivity.java

```diff
@@ -19,20 +19,23 @@
 import com.google.android.gms.tasks.Task;
 import com.google.firebase.auth.AuthResult;
 import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
 
+import uw.studybuddy.FirebaseInstance;
 import uw.studybuddy.MainActivity;
 import uw.studybuddy.R;
 
 public class LoginActivity extends AppCompatActivity{
 
-    private FirebaseAuth mAuth;
+    //private FirebaseAuth mAuth;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         setTheme(R.style.AppTheme_NoActionBar);
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
-        mAuth = FirebaseAuth.getInstance();
+        //mAuth = FirebaseAuth.getInstance();
+        final FirebaseUser user = FirebaseInstance.registerConfirmation().getCurrentUser();
```

Comment:
That instance (user) is not actually used. The snippet of code right under it was using it, but its been commented out for now. Should I just remove it?